Alignment/OfflineValidation/python/TkAlAllInOneTool/plottingOptions.py

```diff
@@ -170,8 +170,6 @@ class PlottingOptionsZMuMu(BasePlottingOptions):
                 "rebineta": "2",
                 "rebinpt": "8",
                 "AutoSetRange": "false",                
-                "CustomMinY": "90.85",
-                "CustomMaxY": "91.5",
```

Comment:
@cardinia I know this PR is closed, but this change in `TkAlAllInOneTool/plottingOptions.py` actually breaks the Z->mumu validation. Can you submit another PR to revert this change?

Reply:
@adewit , those default values were only valid for the Z resonance, and caused a bug when running on the Y resonance.
It's a bug that Rainer found a while ago. There are already default values implemented in the validation tools that depend on the chosen resonance, this change just prevents using the wrong range when running on a different resonance without specifying the range in the config file.

Reply:
Hi @cardinia and @adewit, I've been thinking for a while about how to improve the implementation of X->mumu in the all in one tool.  I think the nicer way to go is to make a base class XMuMuValidation and then make subclasses ZMuMuValidation, Y1SMuMuValidation, etc. and similar for plotting options.  Then the defaults can live in the header and be different for each validation.  It's also more correct in the bookkeeping and would let you run multiple resonances->mumu in the same ini file if you wanted.